doc/classes/TabContainer.xml

```diff
@@ -145,6 +151,7 @@
 			<param index="1" name="icon" type="Texture2D" />
 			<description>
 				Sets the button icon from the tab at index [param tab_idx].
+				[b]Note:[/b] Creates an actuall button that can be pressed. May interfere with the pressing of the tab
```

Comment:
```suggestion
				[b]Note:[/b] Creates an actual button that can be pressed. May interfere with the pressing of the tab.
```

Reply:
As a reader, I'm confused by the term "actual button". What should be the opposite of an "actual button"? A "virtual button"?

As a developer, a tab's button is not an "actual" button (Button node), it's just an image.